transformers/src/unit-test/java/uk/gov/moj/sdt/transformers/UpdateItemTransformerTest.java

```diff
@@ -115,7 +119,7 @@ void testTransformJaxbToDomainResubmitMessage() {
     @Test
     void testTransformJaxbToDomainAccepted() {
         // Set up the jaxb object to transform
-        final String sdtRequestId = "MCOL-12200202-121212";
+        final String sdtRequestId = MCOL_12200202_121212;

```

Comment:
As a constant has been created for this value there's no need for this variable. Remove it and replace instances of it with the constant.

Reply:
replaced

Reply:
Confirmed variable has been removed and replaced with constant.